src/pages/AirTable/AirTable.tsx

```diff
@@ -0,0 +1,58 @@
+import ErrorMessage from "../../components/ErrorMessage/ErrorMessage";
+import Form from "../../components/Form/Form";
+import Loader from "../../components/Loader/Loader";
+import useFetch from "../../hooks/useFetch";
+import styles from "./AirTable.module.css";
+
+export const LS_INPUT_KEY = "inputValue";
+
+const AirTable = () => {
+  const { data, loading, error, refetch } = useFetch(
+    `${process.env.REACT_APP_AIRTABLE}`
+  );
+
+  const formOnSubmit = async (e: Event, name: any) => {
+    localStorage.setItem(LS_INPUT_KEY, name);
+    const newData = await fetch(`${process.env.REACT_APP_AIRTABLE}`, {
```

Comment:
jak masz takiego fajnego hooka odnośnie zapisu w api, to to też tam może powędrować bez problemu